promql/parse.go

```diff
@@ -151,19 +175,16 @@ func (p *parser) typecheck(node Node) (err error) {
 }
 
 // errorf formats the error and terminates processing.
-func (p *parser) errorf(format string, args ...interface{}) {
-	p.error(errors.Errorf(format, args...))
+func (p *parser) errorf(positionRange PositionRange, format string, args ...interface{}) {
```

Comment:
Having an errorf that doesn't have the usual printf/errorf signature is weird.

Reply:
So I should rename it?

Reply:
That'd be best I think